File: HackerHank/MarsExploration/MarsExplorationSolution.py
def marsExploration(s):
    lettersChanged = 0
    i = 0
    while(i < len(s)):
        if('S' != s[i]):
            lettersChanged += 1
        if('O' != s[i+1]):
            lettersChanged += 1
        if('S' != s[i+2]):
            lettersChanged += 1
        i += 3
    return lettersChanged

File: HackerHank/MarsExploration/MarsExplorationSolution.cpp
#include <string>
using namespace std;

int marsExploration(string s)
{
    int lettersChanged;
    for (int i = 0; i < s.size(); i += 3)
    {
        if ('S' != s[i])
            lettersChanged++;
        if ('O' != s[i + 1])
            lettersChanged++;
        if ('S' != s[i + 2])
            lettersChanged++;
    }
    return lettersChanged;
}